GliaWidgets/View/SecureConversations/SecureConversationsWelcomeView.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import UIKit
+
+final class SecureConversationsWelcomeView: View {
+    private let style: SecureConversationsWelcomeStyle
+    let header: Header
+
+    init(with style: SecureConversationsWelcomeStyle) {
+        self.style = style
+        self.header = Header(with: style.header)
+
+        super.init()
+
+        setup()
+        layout()
```

Comment:
`View` class has the necessary methods for settings up and defining UI, so I'd like to propose removing this.
```suggestion
```